<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>804</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1265</width>
    <height>804</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1265</width>
    <height>804</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BOM比较</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>1</y>
      <width>1251</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_New">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>新BOM</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_FileName"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_open">
               <property name="text">
                <string>打开文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_old">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>旧BOM</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_FileName_old"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_open_old">
               <property name="text">
                <string>打开文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_open_cmp">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>比较</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1,5">
             <item>
              <widget class="QLabel" name="label_Change_Befor">
               <property name="text">
                <string>变化前</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::LinksAccessibleByMouse</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_chage">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_Change_After">
               <property name="text">
                <string>变化后</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidgetdiff">
             <column>
              <property name="text">
               <string>物料型号</string>
              </property>
              <property name="font">
               <font>
                <kerning>true</kerning>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>厂家</string>
              </property>
              <property name="foreground">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </property>
             </column>
             <column>
              <property name="text">
               <string>位号</string>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>用量</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>物料型号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>厂家</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>位号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>用量</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>更改类型</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textDirection">
              <enum>QProgressBar::TopToBottom</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>文件保存路径</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_savepath"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_tst">
         <property name="text">
          <string>测试</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
